packages/@coorpacademy-components/src/atom/select-tile/index.js

```diff
@@ -0,0 +1,79 @@
+import React, {useCallback} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import FaIcon from '../icon';
+
+// eslint-disable-next-line css-modules/no-unused-class
+import style from './style.css';
+
+const iconColor = '#0061FF';
+const getButtonContent = (faIcon, selectionMode) => {
+  const selected = selectionMode === 'single' || selectionMode === 'multi';
+  const checkIconNameMap = {
+    single: 'circle-check',
+    multi: 'square-check'
+  };
+  return (
+    <div className={style.contentWrapper}>
+      {selected ? (
+        <div className={style.checkIcon}>
+          <FaIcon
+            iconName={checkIconNameMap[selectionMode]}
+            iconColor={iconColor}
+            backgroundColor={'#ffffff'}
+            size={{faSize: 16, wrapperSize: 16}}
+          />
+        </div>
+      ) : null}
+      <div className={style.iconWrapper}>
+        <div className={style.icon}>
+          <FaIcon
+            iconName={faIcon}
+            iconColor={selected ? iconColor : ''}
+            size={{faSize: 32, wrapperSize: 32}}
+          />
+        </div>
+        <div className={style.iconText}>{faIcon}</div>
+      </div>
+    </div>
+  );
+};
+
+const SelectTile = props => {
+  const {selectionMode, faIcon, 'data-name': dataName, 'aria-label': ariaLabel, onClick} = props;
+
+  const contentView = getButtonContent(faIcon, selectionMode);
+  const styleButton = classnames(
+    style.default,
+    (selectionMode === 'single' || selectionMode === 'multi') && style.selected
```

Comment:
you can store `selectionMode === 'single' || selectionMode === 'multi'`in a "selected" variable for example to reuse it there and in the getButtonContent function